ocs_ci/ocs/constants.py

```diff
@@ -375,7 +375,7 @@
 DEFAULT_NOOBAA_BUCKETCLASS = "noobaa-default-bucket-class"
 NOOBAA_RESOURCE_NAME = "noobaa"
 NOOBAA_DB_PVC_NAME = "db-noobaa-db-pg-0"
-MIN_PV_BACKINGSTORE_SIZE_IN_GB = 17
```

Comment:
we dont need this change. As the issue with pv-pool backingstore creation was because of backticks and its fixed here https://github.com/red-hat-storage/ocs-ci/pull/10048/files